src/server/modes/charts/plugins/ql/js.ts

```diff
@@ -313,7 +313,10 @@ export default ({shared, ChartEditor}: {shared: QlConfig; ChartEditor: IChartEdi
                 distincts: resultDistincts,
             };
         } else {
+            const dataConflict = result.graphs.some((graph: any) => graph.dataConflict);
```

Comment:
You need to add a check for the existence of graphs - in some cases, there may be no data on the chart